src/hub_predtimechart/hub_config.py

```diff
@@ -31,41 +31,47 @@ def __init__(self, hub_dir: Path, ptc_config_file: Path):
 
         self.hub_dir = hub_dir
 
+        # get tasks.json file content
+        with open(self.hub_dir / 'hub-config' / 'tasks.json') as fp:
```

Comment:
Not a today thing, but some of the code below made me realize we should think about how to use or port some of the hub schema reading and parsing functions in packages like `hubAdmin` and `hubUtil`. 

If the Hubverse ever makes a breaking change to the [tasks.json schema](https://github.com/hubverse-org/schemas/blob/main/v3.0.1/tasks-schema.json), any code based on assumptions of its structure could break.

For example: 
```
model_tasks_ele = tasks['rounds'][self.rounds_idx]['model_tasks'][self.model_tasks_idx]
```